ProcessMaker/processmaker
# QA Observation Bugfix/FOUR-17074: Launchpad > Metrics are duplicated

## Solution
- Metrics are not duplicated anymore when Process Info bar is collapsed or uncollpased

## How to Test
- Login to PM
- Go to Processes
- Click on some Process
- Click over Process Info bar many times
- Metrics should not be duplicated.

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-17074

## Code Review Checklist
ci:next
